engine/src/test/scala/cromwell/webservice/CromwellApiServiceSpec.scala

```diff
@@ -140,26 +62,55 @@ class CromwellApiServiceSpec extends FlatSpec with CromwellApiService with Scala
   import scala.concurrent.duration._
 
   // The submit route takes a bit longer than the default 1 s while things initialize, when this spec is run by itself
-  implicit val defaultTimeout = RouteTestTimeout(5.seconds.dilated)
+  implicit val defaultTimeout = RouteTestTimeout(30.seconds.dilated)
 
   override def actorRefFactory = system
+  val summaryActor = system.actorOf(MetadataSummaryRefreshActor.props(None), "metadata-summary-actor")
+
+
   override val workflowManager = actorRefFactory.actorOf(Props(new MockWorkflowManagerActor() with WorkflowDescriptorBuilder {
     override implicit  val actorSystem = context.system
   }))
   val version = "v1"
 
-  // s"CromwellApiService $version" should "return 404 for get of unknown workflow" in {
-  ignore should "return 404 for get of unknown workflow" in {
-    Get(s"/workflows/$version/${MockWorkflowManagerActor.unknownId}") ~>
-      sealRoute(statusRoute) ~>
+  def publishMetadata(events: Seq[MetadataEvent]): Unit = {
+    val timeout: Timeout = 5.seconds.dilated
+    val patienceConfig = PatienceConfig(timeout.duration)
+
+    import akka.pattern.ask
+    val putResult = serviceRegistryActor.ask(PutMetadataAction(events))(timeout)
+    putResult.futureValue(patienceConfig) shouldBe a[MetadataPutAcknowledgement]
+  }
+
+  def forceSummary(): Unit = {
+    val timeout: Timeout = 5.seconds.dilated
+    val patienceConfig = PatienceConfig(timeout.duration)
+    val summaryResult = summaryActor.ask(SummarizeMetadata)(timeout)
+
+    val askResult = summaryResult.futureValue(patienceConfig)
+    askResult match {
+      case MetadataSummarySuccess =>
+      case _ =>
+        fail()
+    }
+
+  }
```

Comment:
These two methods seem like they might be more generally useful than just this test class, and actually I wonder if they might already exist somewhere